routes/meetings.js

```diff
@@ -180,9 +177,7 @@ router
         repeatingCounterIncrement,
         repeatingIncrementBy
       );
-      return res
-        .status(200)
-        .json({ userId: userId, meetingId: newMeeting._id });
+      return res.status(200).json({ userId: userId });
```

Comment:
undo this

Reply:
It will cause issue if I undo that as newMeetings id can also be array of meeting ids

Reply:
cool